metaflow/exception.py

```diff
@@ -30,6 +30,9 @@ def __getstate__(self):
 
     def __setstate__(self, state):
         self.__dict__ = state
+    
+    def __repr__(self):
+        return str(self)
```

Comment:
I think `str` returns the whole backtrace, is that desirable here?

Reply:
@savingoyal What are your thoughts?

Reply:
We can also use `self.type` for `__repr__`;